docker/services/client-api-proxy.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+bash ./docker/build.sh
```

Comment:
having this line in every components script means that dendrite is built one time per component for each time we want to (re)start it. do we really want that?

Reply:
We want that during development to compile the changes. If there isn't something to build, it's practically a no-op.

Reply:
Oh, okay, if gb caches all that then that's fine.